ETL/ETL/_angle.h

```diff
@@ -6,15 +6,20 @@
 ** Copyright (c) 2002 Robert B. Quattlebaum Jr.
 ** Copyright (c) 2007 Chris Moore
 **
-** This package is free software; you can redistribute it and/or
-** modify it under the terms of the GNU General Public License as
-** published by the Free Software Foundation; either version 2 of
-** the License, or (at your option) any later version.
+** This file is part of Synfig.
 **
-** This package is distributed in the hope that it will be useful,
+** Synfig is free software: you can redistribute it and/or modify
+** it under the terms of the GNU General Public License as published by
+** the Free Software Foundation, either version 2 of the License, or
+** (at your option) any later version.
+**
+** Synfig is distributed in the hope that it will be useful,
 ** but WITHOUT ANY WARRANTY; without even the implied warranty of
 ** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-** General Public License for more details.
+** GNU General Public License for more details.
```

Comment:
The word "GNU" repeated twice (see previous line). ^__^

Reply:
Oops :)
I'll fix these cases and rebase.

Reply:
Done. @morevnaproject please review it again :(